configuration/cardano/mainnet-conway-genesis.json

```diff
@@ -0,0 +1,303 @@
+{
+  "poolVotingThresholds": {
+    "committeeNormal": 0.51,
+    "committeeNoConfidence": 0.51,
+    "hardForkInitiation": 0.51,
+    "motionNoConfidence": 0.51,
+    "ppSecurityGroup": 0.51
```

Comment:
@disassembler These are still at `>=51%`. Should we change it to `>50%` (i.e. 50% + Lovelace), by switching it to `9223372036854775808 % 18446744073709551615`?

Reply:
I discussed that with Kevin. The consensus was that the parameter committee voted on 51% because it was assumed you should have a slight more than majority for these settings;. Also, the guardrails require it to be >= 51% as well as a general rule for similar reasons.